common/ir.go

```diff
@@ -30,23 +30,21 @@ func InnerRingInvoker(ir []IRNode) interop.PublicKey {
 
 // InnerRingNodes return list of inner ring nodes from state validator role
 // in side chain.
-func InnerRingNodes() []IRNode {
+func InnerRingNodes() []interop.PublicKey {
 	blockHeight := ledger.CurrentIndex()
-	list := roles.GetDesignatedByRole(roles.NeoFSAlphabet, uint32(blockHeight+1))
-	return keysToNodes(list)
+	return roles.GetDesignatedByRole(roles.NeoFSAlphabet, uint32(blockHeight+1))
 }
 
 // InnerRingNodesFromNetmap gets list of inner ring through
 // calling "innerRingList" method of smart contract.
 // Work around for environments without notary support.
-func InnerRingNodesFromNetmap(sc interop.Hash160) []IRNode {
-	return contract.Call(sc, irListMethod, contract.ReadOnly).([]IRNode)
+func InnerRingNodesFromNetmap(sc interop.Hash160) []interop.PublicKey {
+	return contract.Call(sc, irListMethod, contract.ReadOnly).([]interop.PublicKey)
```

Comment:
After latest change, `irListMethod` returns `[]IRNode`. So we had to convert it into `[]interop.Public.Keys` inside of this method.

Fortunately, it has only two references in whole repository, so it won't be a big overhead. Also `InnerRingNodesFromNetmap` will be removed after enabling notary subsystem.

Reply:
Fixed.